OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -187,6 +189,7 @@ public void onRefresh() {
                 // Hide the progress bar as swipe layout has its own progress indicator
                 loadingIndicator.setVisibility(View.GONE);
                 errorNotification.hideError();
+                canFetchBannerInfo = true;
```

Comment:
why we need to add this `boolean`.

Reply:
currently, CourseBannerAPI is made twice due to code flow, so to handle this issue I used this boolean to make calls only once

Reply:
removed